src/TextareaAutosize.js

```diff
@@ -92,9 +93,18 @@ export default class TextareaAutosize extends React.Component {
     if (maxHeight < this.state.height) {
       props.style.overflow = 'hidden';
     }
+
+    const filteredProps = Object.keys(props).reduce((acc, prop) => {
+      if (blacklistProps.indexOf(prop) === -1) {
```

Comment:
I think filtering better than deleting all unnecessary props from cloned object, it's more readable.
And there is only one right solution, isn't it?)